### ISSUE TITLE
Display Details

### ISSUE THREAD

**Issue by wmreynolds** [heskja/MMM-CalendarWeek#12]
Is there a way to display the details of the calendar events?

**Comment by heskja** [comment]
What kind of details? Location, participants, description or something else?

**Comment by wmreynolds** [comment]
Yeah, like who is participating, the location info and the description that you can put in a google calendar entry.

Very Respectfully,







William Max Reynolds '20

Co-Director of Recruiting Committee

Chair of Logistics and Finance, SPE Sophomore Retreat

Texas A&M University

Society of Petroleum Engineers Student Chapter

Training, Operations, Logistics Sergeant

Texas A&M Corps of Cadets, Infantry Band Staff





Cell: 832.794.0692

wmax.reynolds@hotmail.com



________________________________

**Comment by heskja** [comment]
Location and description should be doable, but I'm a bit uncertain on how to solve this visually, especially if you have limited screen space available. 

Do you have any particular layout in mind?

**Comment by wmreynolds** [comment]
I was thinking to left justify the title of the event, then like bullet it.

So, something like

1630: Dinner with Megan
     - at Perry's Steakhouse (location)
     - remember to buy her flowers (description)

Very Respectfully,







William Max Reynolds '20

Co-Director of Recruiting Committee

Chair of Logistics and Finance, SPE Sophomore Retreat

Texas A&M University

Society of Petroleum Engineers Student Chapter

Training, Operations, Logistics Sergeant

Texas A&M Corps of Cadets, Infantry Band Staff





Cell: 832.794.0692

wmax.reynolds@hotmail.com



________________________________
